uuv_gazebo_worlds/models/ned_frame/model.config

```
@@ -8,7 +8,7 @@
 <model>
   <name>NED frame</name>
   <version>0.0.1</version>
-  <sdf>model.sdf</sdf>
+  <sdf  version='1.6'>model.sdf</sdf>
```

```suggestion
  <sdf version='1.6'>model.sdf</sdf>
```
because my job description says "nitpick to the best of your abilities"

---

Removed space - nitting is an underappreciated practice, thank you.